config/headers.js

```diff
@@ -9,7 +9,7 @@ const securityHeaders = [
   },
   {
     key: 'X-Frame-Options',
-    value: 'SAMEORIGIN',
+    value: 'SAMEORIGIN https://app.contentful.com',
```

Comment:
Also part of a different ticket in the sprint to update to https://github.com/contentful/template-marketing-webapp-nextjs/pull/195/files#diff-29b59c32b97c6884e94911617077b4059b076906b21417f7d61419c9deb5270e?